### ISSUE TITLE
Combine multiple TestOnly stories into a single one with multiple components to reduce snapshot count

### ISSUE THREAD

**Issue by benelan** [Esri/calcite-design-system#7919]
**monday.com sync:** #18048705005

### Test type

chromatic

### Which Component(s)

all

### Additional Info

We should do a pass through our component TestOnly stories looking for ways we can combine multiple stories into a single one, with the goal of reducing our snapshot count. A couple ideas are:

- If there are three stories, one for each `scale`, we can create a single story that contains three components with the different scales
- We can combine stories that are testing different locales into a single `locals_TestOnly` story with multiple components
- Other places where different iterations of the same prop are tested in separate stories can be combined into a single story with multiple components

came up in https://github.com/Esri/calcite-design-system/pull/7915#discussion_r1343105366

**Comment by DitwanP** [comment]
Spike to look into if we still need test changes to reduce snapshot count.

**Comment by isaacbraun** [comment]
Some initial findings/updates:
- `_TestOnly` stories were dropped/renamed via #13554
 - Components that test multiple scales all seem to be combined into one story
 - Components that test multiple locales also seem to combine stories, outside of Input Time Picker which tests 3 locales separately.
 - @benelan's work with TurboSnap should greatly reduce our snapshot count

With those findings, I'm not sure if much more needs to be proactively done to merge stories. We definitely can be aware during PR reviews for opportunities to combine/merge stories, but I may also be missing things.

@benelan, @jcfranco, does this sound right or are there more options I can investigate?

**Comment by jcfranco** [comment]
@isaacbraun I agree with your assessment. Unless, @benelan disagrees, I think we can close this one.

**Comment by benelan** [comment]
Yeah I took a look and I think this can be closed

**Comment by isaacbraun** [comment]
Awesome, thank you both for the feedback! 🤜💥🤛 Closing as not planned based on above discussion.
